packages/module/src/MessageBar/StopButton.tsx

```diff
@@ -0,0 +1,57 @@
+// ============================================================================
+// Chatbot Footer - Message Bar - Send
+// ============================================================================
+import React from 'react';
+
+// Import PatternFly components
+import { Button, ButtonProps, Tooltip, TooltipProps, Icon } from '@patternfly/react-core';
+
+export interface StopButtonProps extends ButtonProps {
+  /** Callback for when button is clicked */
+  onClick: (event: React.MouseEvent<HTMLButtonElement>) => void;
+  /** Class name for StopButton */
+  className?: string;
+  /** Props to control the PF Tooltip component */
+  tooltipProps?: TooltipProps;
+  /** English text "Stop" used in the tooltip */
+  tooltipContent?: string;
+}
+
+export const StopButton: React.FunctionComponent<StopButtonProps> = ({
+  className,
+  onClick,
+  tooltipProps,
+  tooltipContent = 'Send',
```

Comment:
Should the tooltip content be "stop" instead?

Reply:
Bug alert! Thank you.